java/src/test/java/com/saucelabs/simplesauce/BaseTestConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.saucelabs.simplesauce;
+
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({System.class})
+class BaseTestConfiguration {
+    protected SauceSession sauce;
+    protected SauceOptions sauceOptions;
+    private SauceRemoteDriver dummyRemoteDriver;
+
+    @Before
+    public void setUp() {
+        dummyRemoteDriver = mock(SauceRemoteDriver.class);
+        sauceOptions = new SauceOptions();
+
+        sauce = new SauceSession();
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.getenv(Mockito.eq("SAUCE_USERNAME"))).thenReturn("test-name");
+        PowerMockito.when(System.getenv(Mockito.eq("SAUCE_ACCESS_KEY"))).thenReturn("accessKey");
+        PowerMockito.when(System.getenv(Mockito.any())).thenCallRealMethod();
+
+        sauceOptions.getBrowserName();
+    }
+
+    SauceSession instantiateSauceSession() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4730520)